tests/test_resolver.py

```diff
@@ -26,3 +50,99 @@ def test_open_stage_and_logging(capfd):
     assert "_OpenAsset" in outputs[6]
     assert "_GetModificationTimestamp" in outputs[7]
     assert "_GetExtension" in outputs[8]
+
+
+# As all the data tends to follow the same form, convenience method
+# to avoid repeating myself
+def assert_parking_lot_structure(usd_stage):
+    floor = usd_stage.GetPrimAtPath("/ParkingLot/ParkingLot_Floor_1")
```

Comment:
[fix] I think itd be worth being explicit and adding `assert floor.IsValid()`, etc. So we're not implicitly relying on the specifics of the erst of the test.

Reply:
https://github.com/OpenAssetIO/usdOpenAssetIOResolver/pull/13/commits/3dbf21674ddde2760055681f4de77e64395d3b49